apps/system/js/bluetooth_transfer.js

```diff
@@ -530,6 +544,42 @@ var BluetoothTransfer = {
       detail: details}));
   },
 
+  /**
+   * Handle related system notification message operation after reboot.
+   * Need close notification message by ourself.
+   *
+   * @param {Object} message system notification message
+   */
+  handleSystemMessageNotification: function(message) {
+    this.debug('Received system message: ' + JSON.stringify(message));
+    if (message.data && message.data.transferInfo) {
+      var evt = {
+        fileName: message.data.transferInfo.fileName,
+        contentType: message.data.transferInfo.contentType
+      };
+      this.openReceivedFile(evt);
+    } else {
+      console.error('The notification does not carry correct data');
+    }
+    this.closeSystemMessageNotification(message);
```

Comment:
we can't do anything if bucket is leaked, at least we could make it act like other bluetooth notification message (click to close)